README.md

```diff
@@ -125,12 +125,46 @@ This will allow you to easily map Gutenberg blocks to front-end components. Post
 
 See our [Next.js boilerplate][nextjs-boilerplate] for an example of how to use and render these blocks.
 
+*Note: In a future update, this plugin will be deprecated and eventually removed in favour of the VIP Block Data API plugin.*
+
 ### WPGraphQL Preview
 
 This plugin overrides WordPress's native preview functionality and securely sends you to your decoupled frontend to preview your content. This ensures that your preview content has parity with your published content. It works by issuing a one-time use token, locked to a specific post, that can be redeemed by the frontend to obtain preview content for that post.
 
 **This plugin currently only works with our Next.js boilerplate** and should be disabled if you are not using it. If you are interested in using this plugin for other frontend frameworks, please see the [preview `README`][preview-readme].
 
+### VIP Block Data API
```

Comment:
You might want to put both options next to each other and add a "Which should I choose?" section. That section can even just say, "we already recommend the block data API but here are some caveats"

Reply:
Great point, thanks for pointing that out. I've added this, and added what I think would be helpful in making this decision. Lemme know what you think.